### PR TITLE
Theme tukan/cicada - short tokenize2 fix

### PR DESCRIPTION
**Pull request by opnsenseuser**

### CODE REVIEW

**Comment by opnsenseuser** [comment]
@fichtner would be great if you could merge these liitle fixes before next release.. thx regards rene

File: misc/theme-tukan/src/opnsense/www/themes/tukan/assets/stylesheets/tokenizer2.scss [hunk 1]
```diff
@@ -159,4 +159,4 @@
       }
     }
   }
-}
+}
```
Comment (fichtner) [inline misc/theme-tukan/src/opnsense/www/themes/tukan/assets/stylesheets/tokenizer2.scss]:
```suggestion
}
```

Comment (opnsenseuser) [reply misc/theme-tukan/src/opnsense/www/themes/tukan/assets/stylesheets/tokenizer2.scss]: Sorry that i forgot the last line. Hope you can merge it now and i didnt forget anything else.

Comment (fichtner) [reply misc/theme-tukan/src/opnsense/www/themes/tukan/assets/stylesheets/tokenizer2.scss]: you need to accept/commit the suggestion ;)

Comment (opnsenseuser) [reply misc/theme-tukan/src/opnsense/www/themes/tukan/assets/stylesheets/tokenizer2.scss]: can´t accept or commit!

Comment (opnsenseuser) [inline misc/theme-tukan/src/opnsense/www/themes/tukan/assets/stylesheets/tokenizer2.scss]: can´t accept only mark as resolved

Comment (opnsenseuser) [reply misc/theme-tukan/src/opnsense/www/themes/tukan/assets/stylesheets/tokenizer2.scss]: grafik

**Comment by fichtner** [comment]
Strage, will merge and fix then. Thanks!

**Pull request merged**